docs/participate/online-communities.mdx

```diff
@@ -8,46 +8,34 @@ The Mina community has many online groups where you can connect with other Mina
 
 ## Discord
 
-The [Mina Protocol Discord](https://discord.com/invite/RDQc43H) is the most popular place where Mina enthusiasts and technical contributors gather. Some popular channels are listed below, but many more channels exist, so explore the server to find one that fits your interest.
+[Mina Protocol Discord](https://discord.gg/minaprotocol) is the most popular place where Mina enthusiasts and technical contributors gather. Some popular channels are listed here, but many more channels exist, so explore the server to find one that fits your interest.
 
-[#announcements](https://discord.gg/JRU3PENEB2) — to follow announcements
-
-[#zkapps-general](https://discord.gg/MVfuuEvD4U) — to ask general questions about zkApps, how to use a zkApp, etc.
-
-[#zkapps-developers](https://discord.gg/v3W4YsuBCw) — to meet other developers building zkApps with SnarkyJS
-
-[#mentor-nodes](https://discord.gg/j9GHr7pJrR) — to get help from the community on setting up and running nodes
-
-[#mainnet-block-producer](https://discord.gg/eAD6V4wW7N) — to meet other node operators and block producers
-
-[#grants](https://discord.gg/AcFCvVvfVW) — to discuss grant opportunities
+- **#announcements** to follow announcements
```

Comment:
I am leaning strongly to remove the channel-specific URLs since the possibility of account verification friction is very high (almost 100%)

When you are already logged in to the Discord .com/app on a web browser, you are still directed to relogin... and it's very likely that you will create a phantom account (as many of us have)

I suggest that we just direct to the main Discord URL and let users navigate to channels directly, wdyt @iregina 

let's test it together!

Reply:
@barriebyron Based on your recent DM's we can go with channel-specific evergreen link correct?